src/titan_checkpoint_test.cc

```diff
@@ -139,7 +139,7 @@ class CheckpointTest : public testing::Test {
 
   void Close() {
     for (auto h : handles_) {
-      delete h;
```

Comment:
This does not follow the API protocol. And the new change breaks these tests due to the complicity of handling `default_cf_handle_`'s lifetime.